neqo-bin/src/server/http09.rs

```diff
@@ -10,7 +10,7 @@ use neqo_common::{event::Provider, hex, qdebug, qerror, qinfo, qwarn, Datagram};
 use neqo_crypto::{generate_ech_keys, random, AllowZeroRtt, AntiReplay};
 use neqo_http3::Error;
 use neqo_transport::{
-    server::{ActiveConnectionRef, Server, ValidateAddress},
+    server::{ConnectionRef, Server, ValidateAddress},
```

Comment:
As an aside, this pull request renames `ActiveConnectionRef` to `ConnectionRef`. I don't see why the `Active` prefix is needed. In other words, why would the connection not be active? Not a strong opinion. Happy to revert.